web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/formidling/Dataset.java

```diff
@@ -0,0 +1,24 @@
+package no.nav.foreldrepenger.web.app.tjenester.formidling;
+
+public enum Dataset {
+    FAGSAK,
```

Comment:
Hva er tanken med disse? Inspirert av abakus? Abakus vet ytelse, men ikke BehType.
Her kaller du fpsak og den vet mye mer - Kan du ikke alltid ta med sak+behandling+verge + gjøre noen betinget av ytelse, behandlingtype (klage vs ytelse), behandlingResultat (avslag = droppe en del beregning, uttak, tilkjent)

Reply:
Det er i stor grad det samme som dagens integrasjon gjør også. Det er nødvendig å ha en mekanisme for å styre hva som skal leveres tilbake fra fp-sak og sammenligne responsen med svaret fra det gamle API-et – tenk utvidelse av kontrakt.

Etter overgangen til et eget API kan vi vurdere fase 2 og mulige forenklinger.

Reply:
Nå er det slik at fpformidling kaller ulike endepunkt basert på utledning og ressurslenker.
Legger du opp til en maxi-dto som  populeres basert på dataset ? 
Klassisk sett har delvis utfylte maxi-dtos ført til at man har knekt de opp i flere dtos og endepunkt. 
Er det ikke like greit å gå for en fullstendig maxiDto - alltid utfylt - så kan formidling bruke det den trenger?

Reply:
Jeg har ikke gått i detalj på hvert enkelt brev ennå – først må jeg forstå databehovet for å kunne ta en avgjørelse.

Målet med denne oppgaven er å bygge et eget API for formidling, der vi i første omgang sannsynligvis bare gjenbruker DTO-ene som de er, eventuelt med noen mindre forenklinger. Disse skal i starten kun legges inn i BrevGrunnlagDto. Neste fase vil fokusere på videre forenkling.

Reply:
Bra tiltak dersom du får til å skille endepunkt / Dtos som formidling bruker helt fra de som frontend bruker. Nå er det en del egne Dtos, men fortsatt noen mixed-bruk.
Så er det litt styr med disse "fagsak/verge-backend" lenkene som brukes av formidling og tilbake. Det er antagelig best med 3 sett lenker/dtos - frontend, formidling, tilbake